### PR TITLE
Changed to use tomcat-base (CADC-13798)

### PR DESCRIPTION
**Pull request by andamian**

### CODE REVIEW

File: src/main/webapp/service.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          in: query
+          description: 'Name of the object to resolve. Examples: target=M31'
+          required: true
+          type: string
+        - name: service
+          in: query
+          description: 'Services to query. Examples: service=NED'
+          required: false
+          default: ALL
+          values:
```
Comment (at88mph) [inline src/main/webapp/service.yaml:63]: The `values` property is not allowed.  This should be `enum`.

File: src/main/webapp/service.yaml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            - ALL
+          type: string
+          collectionFormat: multi
+        - name: format
+          in: query
+          description: 'Response FORMAT. Examples: format=JSON'
+          required: false
+          type: string
+          default: ASCII
+          values:
```
Comment (at88mph) [inline src/main/webapp/service.yaml:76]: The `values` property is not allowed.  This should be `enum`.

File: src/main/webapp/service.yaml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            - ASCII
+            - JSON
+        - name: cached
+          in: query
+          description: 'Whether to return previous cached results. Examples: cached=YES'
+          required: false
+          type: boolean
+          default: YES
+          values:
```
Comment (at88mph) [inline src/main/webapp/service.yaml:85]: The `values` property is not allowed.  This should be `enum`.

File: src/main/webapp/service.yaml [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+            - YES
+            - NO
+        - name: detail
+          in: query
+          description: 'Details to return. Examples: detail=MAX'
+          required: false
+          type: string
+          default: MIN
+          values:
```
Comment (at88mph) [inline src/main/webapp/service.yaml:94]: The `values` property is not allowed.  This should be `enum`.

**Review by at88mph — CHANGES_REQUESTED** [review]

File: VERSION [hunk 5]
```diff
@@ -0,0 +1,7 @@
+## deployable containers have a semantic and build tag
+# semantic version tag: major.minor[.patch]
+# build version tag: timestamp
+# tag: {semantic}-{build}
+VER=1.0.1
+TAGS="${VER} ${VER}-$(date --utc +"%Y%m%dT%H%M%S")"
```
[the remark this answers is not in the document]
Comment (at88mph) [reply VERSION:6]: Can you please make this one last change as well?

**Review by at88mph — APPROVED** [review]

File: VERSION (hunk 5 above)
[the remark this answers is not in the document]
Comment (andamian) [reply VERSION]: Done - thanks

**Pull request merged**